php/WP_CLI/Formatter.php

```diff
@@ -356,12 +356,28 @@ public function transform_item_values_to_json( $item ) {
 		foreach ( $this->args['fields'] as $field ) {
 			$true_field = $this->find_item_key( $item, $field );
 			$value      = is_object( $item ) ? $item->$true_field : $item[ $true_field ];
+
+			// Transform into JSON.
 			if ( is_array( $value ) || is_object( $value ) ) {
 				if ( is_object( $item ) ) {
 					$item->$true_field = json_encode( $value );
 				} elseif ( is_array( $item ) ) {
 					$item[ $true_field ] = json_encode( $value );
 				}
+
+				return $item;
```

Comment:
Is it safe to return here inside the `foreach` loop?

Reply:
@BrianHenryIE You're correct. This `return` would definitely cause issues in other cases. The original idea was to return as soon as the value type is known, so we wouldn't have to run it through another condition to check the type. So I've updated the spec and the code to handle scenarios where it might fail.

Thank you for your feedback :)